playwright.config.ts

```diff
@@ -0,0 +1,32 @@
+import { devices, PlaywrightTestConfig } from '@playwright/test';
+
+// Config file docs: https://playwright.dev/docs/test-configuration
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: 'pnpm start',
+    port: 3000,
+    reuseExistingServer: !process.env.CI,
+  },
```

Comment:
To launch a development web server during the tests. It specifies the `command` to start the server, the `port` number, and whether to reuse an existing server (`reuseExistingServer`).
https://playwright.dev/docs/api/class-testconfig#test-config-web-server

Reply:
I think we can remove `reuseExistingServer` - it creates inconsistent behavior where locally, Playwright will not fail if there's a server already running

two potential problems I thought of:

1. student runs something else on port 3000 (test will fail from other application not fulfilling the test conditions)
2. student runs dev server on port 3000 (test may fail from inconsistencies between development and production environments)

Reply:
looks like `reuseExistingServer` was removed in some commit, although not commented here

Reply:
```
command: 'pnpm start',
```

should this be changed at all based on the research below?

- https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/issues/10

Reply:
Changing the command from `pnpm start` to `/node_modules/.bin/next start` will not make any difference if we only test on the build server. The local build server will already have all of the dependencies installed, and there is no need to save memory.

Reply:
- it doesn't have to do with dependencies being installed - it's about running with lower RSS (Resident Set Size)
- we should minimize this in every environment, always

Reply:
Updated the `webServer` command 
[98d4078](https://github.com/upleveled/next-js-portfolio-web/pull/107/commits/98d40784ec53548fc32e1f74e66d11fb0401374f)

Reply:
ok, the research hasn't been yet completed in https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/issues/10, but to get this merged quicker, we can put this in with the `./node_modules/.bin/next start` suggestion for now

if we decide to change to `output: 'standalone'` in the future, then we need to change all of the `./node_modules/.bin/next start ` occurrences back